main.go

```diff
@@ -219,41 +219,37 @@ func apply(configFile string) (ctrl.Options, config.Configuration) {
 	options := ctrl.Options{
 		Scheme: scheme,
 	}
+	cfg := config.Configuration{}
 
-	config := config.Configuration{}
-
+	successLoadConfigurationMessage := "Successfully loaded configuration"
 	if configFile == "" {
```

Comment:
I meant something more like this (take it as pseudocode)

```
if configFile == "" {
  scheme.Default(cfg)
  options = options.AndFrom(cfg)
} else {
  options = options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(cfg))
}
cfgStr = encodeConfig(cfg)
setupLog("Successfully loaded configuration", "config", cfgStr))
```

Reply:
Thanks for clarifying!

Reply:
Done.